### DISCUSSION TITLE
What is this problem

### DISCUSSION THREAD

**Discussion by mnz12138** [WalletConnect/walletconnect-monorepo#2415] [Developer Support]
https://github.com/WalletConnect/walletconnect-monorepo/assets/13588391/6b5164a5-6514-4005-bcfa-e2741d8f159e

**Comment by finessevanes** [comment]
Hey - can you share some details on the implementation? Please include what SDKs and versions you are also using.

**Comment by mnz12138** [comment]
Download WalletConnectSwiftV2-Example-DApp, Modified connectWithExampleWallet method, let url = URL(string: "rainbow://wc?uri=\(uri.absoluteString)")!

**Comment by finessevanes** [comment]
Can you walk me through the steps to reproduce?

**Comment by mnz12138** [comment]
This is the demo I'm using so far
点按以下载
 <https://www.icloud.com/attachment/?u=https%3A%2F%2Fcvws.icloud-content.com.cn%2FB%2FAa_LviEX47aPZVQsPEuxtVFZZP0CAXNfjOker_mQtMQArY4KCMTI7psv%2F%24%7Bf%7D%3Fo%3DAvpld7MO0ns1PpzFdXwMqWmNflxoOaOPOEsAu859sWav%26v%3D1%26x%3D3%26a%3DCAogb9Y68suNhRfJdbh7bpglhI3DOP5o-VU3p-v9CFKCxxESehC43YqXgzEYuO2F64wxIgEAKgkC6AMA_1B0FGBSBFlk_QJaBMjumy9qJ-4TSaV6fLiITpcoaCkR7qZscMvY7lluFw4GmkR3JxTxCt_u9ue0A3In2eXhi6p6idNHZBxZ5Mdv_6hACfgIzRHtK-ciUhXyl8yhM-mNo1Cv%26e%3D1687072896%26fl%3D%26r%3D9CD5CEAA-0091-4315-BFD8-CDB7ACD4EEF2-1%26k%3D%24%7Buk%7D%26ckc%3Dcom.apple.largeattachment%26ckz%3D551AF5D2-A1C0-48F1-B63D-5EB4B2DF0AF4%26p%3D216%26s%3DWkT7PeSllNWXSCyFhwQqL32eHi8&uk=Ta5-ZHRQt5bwLy2rwPvc9A&f=WalletConnectSwiftV2.zip&sz=60595403>WalletConnectSwiftV2.zip
60.6 MB

**Comment by mnz12138** [comment]
附件将于2023年6月18日过期
This is the demo I'm using so far
点按以下载
 <https://www.icloud.com/attachment/?u=https%3A%2F%2Fcvws.icloud-content.com.cn%2FB%2FAa_LviEX47aPZVQsPEuxtVFZZP0CAXNfjOker_mQtMQArY4KCMTI7psv%2F%24%7Bf%7D%3Fo%3DAvpld7MO0ns1PpzFdXwMqWmNflxoOaOPOEsAu859sWav%26v%3D1%26x%3D3%26a%3DCAogb9Y68suNhRfJdbh7bpglhI3DOP5o-VU3p-v9CFKCxxESehC43YqXgzEYuO2F64wxIgEAKgkC6AMA_1B0FGBSBFlk_QJaBMjumy9qJ-4TSaV6fLiITpcoaCkR7qZscMvY7lluFw4GmkR3JxTxCt_u9ue0A3In2eXhi6p6idNHZBxZ5Mdv_6hACfgIzRHtK-ciUhXyl8yhM-mNo1Cv%26e%3D1687072896%26fl%3D%26r%3D9CD5CEAA-0091-4315-BFD8-CDB7ACD4EEF2-1%26k%3D%24%7Buk%7D%26ckc%3Dcom.apple.largeattachment%26ckz%3D551AF5D2-A1C0-48F1-B63D-5EB4B2DF0AF4%26p%3D216%26s%3DWkT7PeSllNWXSCyFhwQqL32eHi8&uk=Ta5-ZHRQt5bwLy2rwPvc9A&f=WalletConnectSwiftV2.zip&sz=60595403>WalletConnectSwiftV2.zip
60.6 MB
